strawberryfields/engine.py

```diff
@@ -369,21 +364,17 @@ def _run_program(self, prog, **kwargs):
                 ) from None
         return applied
 
-    def run(self, program, *, args=None, compile_options=None, run_options=None):
+    def run(self, program, *, args=None, compile_options=None, **run_options):
```

Comment:
Wondering here: should the `run_options` = keyword arguments be mentioned in the docstring?

Reply:
I'm wondering as well. `Keyword Args:` is part of the sphinx.napoleon extension though, so I left is as that in the end 🤔

Reply:
Sure I think that makes sense. I think it's fine to leave it as is or to add a (slightly unconventional) note to the beginning of the section on this. For me either works fine :)